client/pages/event-details.jsx

```diff
@@ -9,39 +16,55 @@ export default class EventDetails extends React.Component {
   }
 
   componentDidMount() {
-    fetch(`https://api.seatgeek.com/2/events/${this.props.eventId}`)
+    fetch(`https://api.seatgeek.com/2/events/${this.props.eventId}?client_id=MjQyNjE3MTd8MTYzNTk2NTg3MS40NzI3Mjg3`)
```

Comment:
Assuming `client_id` is an API key, the actual key should be imported from a `.gitignore`d file so you can use a different key on each deployment (local development vs live site).